python/SFProvider.py

```diff
@@ -4,6 +4,8 @@ class SFProvider:
     supported_decay_modes = [ 0, 1, 10, 11 ]
 
     def __init__(self, input_file, channel, wp):
+        if 'UL' in input_file:
+            SFProvider.supported_decay_modes = [ 0, 1, 10, 11, 1011]
```

Comment:
I would not leave the decision on using the decay modes 10 and 11 separately or combined to the user. We do not recommend the usage of decay modes 10 and 11 separately. So we should internally map these decay modes to the 1011 histogram. In this way, the user does not need to map the decay modes beforehand.

Reply:
Hi, I thought we may need the DM 10 and DM 11 separately, because when we see at the mu-tauh rich samples the statistics are fine enough for DM 10 and 11.  Okay, now corrected that, If anyone could remove this PR I could ask for the new PR.